app/_data/tables/version_errors_konnect.yml

```
@@ -4208,4 +4270,165 @@ messages:
     Resolution: |
       Please upgrade Kong Gateway to version `3.9.0.0` or above.
     DocumentationURL: |
-      N/A
+
+  - ID: P309
+    Severity: error
+    Description: |
+      One or more streaming custom plugin entities have been configured, but streaming
+      support is not enabled on the data plane node. Streaming custom plugins have
+      been removed from the config and the data plane may be inoperable.
+    Resolution: |
+      Enable streaming custom plugin support on the data plane node or remove
+      custom-plugin entities from the config.
+    DocumentationURL: |
+      [Custom_plugin plugin](/hub/kong-inc/custom_plugin/)
```

This URL doesn't exist. Info about streaming plugins will be here (not live yet):
```suggestion
      [Streaming custom plugins](/gateway-manager/dedicated-cloud-gateways/custom-plugins/)
```